### PR TITLE
Add Perl 6 topic

### PR DESCRIPTION
**Pull request by hankache**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

These changes are necessary so that perl6 becomes a topic on its own instead of an alias to perl.  
In addition a typo has been corrected.

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

Perl 6 is considered a new language rather than an upgrade to Perl. As an analogy think of C and C++. Thus the need to have a separate topic for it.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by hankache** [comment]
The failing test is related to this guideline "Write out “one” and every number less than 10" and the language name contains a number: "Perl 6". Any idea on how to fix this?

**Comment by cheshire137** [comment]
Oh no, I see the test failure. I think we could add an exception in the test to ignore "Perl 6" specifically, since that's the proper name. This worked in my local testing:

```diff
+ end

- numbers_to_be_spelled_out.each do |digit|
-   refute_includes line, " #{digit}#{punctuation}",
-                   'Write out "one" and every number less than 10'
- end
+ match = line.match(/\b(\w+)\s\d[.,;:\s]/)
+ if match
+   allowed_words_before_numbers = %w[Perl]
+   assert_includes allowed_words_before_numbers, match[1],
+                   'Write out "one" and every number less than 10, except when they ' +
+                   "follow one of: #{allowed_words_before_numbers.join(', ')}"
+ end
```

This is in topics_test.rb starting at line 319.

File: topics/perl6/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+github_url: https://github.com/perl6
+logo: perl6.png
+related: language, perl
+released: December 25, 2015
+short_description: Perl 6 is an expressive and feature-rich programming language.
+topic: perl6
+url: https://www.perl6.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Perl_6
+---
+Perl 6 is an expressive and feature-rich programming language designed by Larry Wall and developed by the Perl 6 community. Features include: Object-oriented programming, functional programming primitives, parallelism, concurrency, asynchrony, definable grammars for pattern matching and generalized string processing, optional and gradual typing.
```
Comment (cheshire137) [inline topics/perl6/index.md]: Could you update this to:

- Lowercase "Object" in "include: Object-oriented"
- Add "and" before "optional and gradual typing"

Comment (hankache) [reply topics/perl6/index.md]: @cheshire137 Done. Thanks a lot.

**Pull request merged**